jupyter-docker/jupyter_notebook_config.py

```diff
@@ -14,5 +14,5 @@
 c.NotebookApp.allow_origin = '*'
 
 fragment = os.environ['GOOGLE_PROJECT'] + '/' + os.environ['CLUSTER_NAME']
-c.NotebookApp.base_url = '/api/notebooks/' + fragment + '/'
-c.NotebookApp.webapp_settings = {'static_url_prefix':'/api/notebooks/' + fragment + '/static/'}
\ No newline at end of file
+c.NotebookApp.base_url = '/notebooks/' + fragment + '/'
```

Comment:
@ansingh7115 we'll need to rebuild the Jupyter docker image yet again. Is this process documented somewhere so I don't have to keep bugging you to do it?

(I might have yet another change for GAWB-2535 -- TBD)

Reply:
Maybe a README.md under jupyter-docker/ would be helpful

Reply:
Or a `build.sh`